src/components/organisms/ProfileModal/EditProfileForm/EditProfileForm.tsx

```diff
@@ -68,64 +75,69 @@ const EditProfileForm: React.FunctionComponent<PropsType> = ({
   const pictureUrl = watch("pictureUrl");
 
   return (
-    <div className="edit-profile-modal">
-      <h1 className="title">Edit profile</h1>
-      <form onSubmit={handleSubmit(onSubmit)} className="form">
-        <div className="input-group profile-form">
-          <input
-            name="partyName"
-            className="input-block input-centered"
-            placeholder="Your display name"
-            ref={register({
-              required: true,
-              maxLength: DISPLAY_NAME_MAX_CHAR_COUNT,
-            })}
+    <div className="EditProfileForm">
+      <h1 className="EditProfileForm__title">Edit profile</h1>
+      <div className="form">
+        <InputField
+          containerClassName="EditProfileForm__input-container-name"
+          inputClassName="EditProfileForm__input-name"
+          name="partyName"
+          placeholder="Your display name"
+          ref={register({
+            required: true,
+            maxLength: DISPLAY_NAME_MAX_CHAR_COUNT,
+          })}
+        />
+        {errors.partyName && errors.partyName.type === "required" && (
```

Comment:
```suggestion
        {errors.partyName?.type === "required" && (
```

Reply:
Is it possible that the error types are exported from the library?

Reply:
If not, we should create an enum for those

Reply:
Oh, since it was copyied, NB

Reply:
added `Yup` validation here: 
https://github.com/sparkletown/sparkle/pull/1499/commits/4b9c4fee3c3c8ac3d75c4a18814995447bfa29da#diff-9456a36066fb33e91efde7cd544a40ca21b20f06174f7bf2a401eb0da7f1e7e1R28-R37

removed these blocks: https://github.com/sparkletown/sparkle/pull/1499/commits/4b9c4fee3c3c8ac3d75c4a18814995447bfa29da#diff-9456a36066fb33e91efde7cd544a40ca21b20f06174f7bf2a401eb0da7f1e7e1L91-L99
(because I have improved `InputField` to accept the input-error and show the error-message.)